shared/helpers/assertRevert.js

```diff
@@ -39,6 +39,7 @@ module.exports = web3 => {
 
   async function assertRevertGanache(request, reason) {
     const tx = request.params[0]
+    if (!tx.from) tx.from = web3.eth.accounts[0]
```

Comment:
The problem was that since we were using the `contract.method.request` way to build a tx in order to assert it reverts, the `from` field wasn't being set, then this is basically mimicking what truffle does